src/api/BoreholeLockService.cs

```diff
@@ -87,6 +87,22 @@ public bool HasUserWorkgroupPermissions(Borehole borehole, User user)
         return user.WorkgroupRoles != null && user.WorkgroupRoles.Any(x => x.WorkgroupId == borehole.WorkgroupId);
     }
 
+    /// <inheritdoc />
+    public async Task<bool> IsUserLackingWorkgroupRoleAsync(string? subjectId, int workgroupId, Role expectedRole)
+    {
+        var user = await GetUserWithWorkgroupRolesAsync(subjectId).ConfigureAwait(false);
+        return IsUserLackingWorkgroupRole(user, workgroupId, expectedRole);
+    }
+
+    /// <inheritdoc />
+    public bool IsUserLackingWorkgroupRole(User user, int workgroupId, Role expectedRole)
+    {
```

Comment:
Ich denke wenn der User Admin ist, darf er in jedem Fall importieren.
```suggestion
    {
    if (user.IsAdmin) return true;
```